docs/LICENSE_OF_DEPENDENCIES.md

```diff
@@ -115,6 +115,7 @@ following works:
 - github.com/gosnmp/gosnmp [BSD 2-Clause "Simplified" License](https://github.com/gosnmp/gosnmp/blob/master/LICENSE)
 - github.com/grid-x/modbus [BSD 3-Clause "New" or "Revised" License](https://github.com/grid-x/modbus/blob/master/LICENSE)
 - github.com/grid-x/serial [MIT License](https://github.com/grid-x/serial/blob/master/LICENSE)
+- github.com/gwos/tcg/sdk [TCG License](https://github.com/gwos/tcg/blob/master/LICENSE)
```

Comment:
TCG License? It looks like GNU GPL and we cannot accept this one as it implies telegraf will need to change its license as well..

Reply:
I'll ask Thomas Stocking about it.

Reply:
Please update this information to the new license.